packages/mongodb-mql-model/src/main/kotlin/com/mongodb/jbplugin/mql/components/HasValueReference.kt

```diff
@@ -18,7 +18,7 @@ data class HasValueReference(
      * @property type
      */
     data class Constant(
-        val value: Any,
+        val value: Any?,
```

Comment:
Is this nullable because a Constant can be null? Or is it something else? Ideally Constant values can always be resolved, if they can not, we would use a Runtime value.

Reply:
yea exactly because the constant itself can be null. There was a small parsing error that [fix for nullable references getting detected as BsonAny instead of Bs…](https://github.com/mongodb-js/intellij/pull/33/commits/c698b7249e18ff75e6cc3790ea5cc73bc16ce1e0) fixes and it was related to constant null being parsed as BsonAny